gradle/libs.versions.toml

```diff
@@ -207,7 +219,7 @@ kotlinx-coroutines-rx2 = { module = "org.jetbrains.kotlinx:kotlinx-coroutines-rx
 kotlinx-coroutines-slf4j = { module = "org.jetbrains.kotlinx:kotlinx-coroutines-slf4j", version.ref = "kotlin-coroutines" }
 kotlinx-coroutines-reactor = { module = "org.jetbrains.kotlinx:kotlinx-coroutines-reactor", version.ref = "kotlin-coroutines" }
 
-log4j = { module = "org.apache.logging.log4j:log4j-core", version.ref = "log4j" }
+log4j = { module = "org.apache.logging.log4j:log4j-core", version.ref = "managed-log4j" }
```

Comment:
:x: This should be managed too, right?

Reply:
@altro3 we moved versions outside of core intentionally in 4.0.x. I think `micronaut-logging` is probably the way to go.

Reply:
@sdelamo good solution! But where is project micronaut-logging? :-)